packages/webpack-utilities/src/fabricAsyncLoader.ts

```diff
@@ -4,8 +4,8 @@ import * as loaderUtils from 'loader-utils';
 function getMagicComments(options: loaderUtils.OptionObject): string {
   const magicCommentsOptions: { [key: string]: string } = {
     webpackChunkName: `"${options.chunkName || 'fabric.async'}"`,
-    webpackPrefetch: options.prefetch,
-    webpackPreload: options.preload,
+    webpackPrefetch: options.prefetch as string,
+    webpackPreload: options.preload as string,
```

Comment:
In `@types/loader-utils@1.1.3` all options were `any`:

```ts
// @types/loader-utils@1.1.3
export interface OptionObject {
    [key: string]: any;
}

// @types/loader-utils@2.0.0
export interface OptionObject {
    [key: string]: null | false | true | string;
}
```

---

_Anyway, it's not a good idea to use such type for options_ 😕